### ISSUE TITLE
page up/down miss some lines

### ISSUE THREAD

**Issue by PhilLAL** [audreyt/ethercalc#358]
Hi, 
when I go up/down with keyboard page up/down buttons, this goes over the last visible cell.
I mean when I see line 1 to 20 and I click page down, I see line 25 to further, and so I miss line 21 to 24.
It seems to me it could be due to variable height lines.

A good behavior would be to slide exactly to the next visible line when click on page up/down.
Is there a mean to do this ?

**Comment by audreyt** [comment]
Yes, it is because PgUp/PgDn is currently hard-coded to move 15 rows. (`socialcalcconstants.js` line 216 and `socialcalctableeditor.js` line 169)... Pull requests welcome.
